### ISSUE TITLE
prop 'borderStyle' is not working

### ISSUE THREAD

**Issue by LucasHimelfarb** [colbymillerdev/react-native-progress-steps#23]
prop 'borderStyle' is not working for < ProgressSteps props... />

**Comment by colbymillerdev** [comment]
This seems to be a known issue with React Native and border styles. I'll continue to look into this to see if I can find a good solution.

**Comment by CostaPedroHenrique** [comment]
Alguma atualização?

**Comment by LucasHimelfarb** [comment]
No, you can try make a components with simple logic for this :)
